apps/notifications/emails.py

```diff
@@ -1,10 +1,44 @@
+from email.mime.image import MIMEImage
+
+from django.contrib.staticfiles import finders
+
 from adhocracy4 import emails
 
 
-class SubmitNotification(emails.UserNotification):
+class SVGLogoMixin:
```

Comment:
While I like SVG really a lot, it might not be the best option for sending in emails. For sure it is much smaller, but many emails might be read on [gmail mobile or outlook](https://css-tricks.com/a-guide-on-svg-support-in-email/).